src/components/TabMenu/TabMenu.js

```diff
@@ -83,10 +92,18 @@ class TabMenu extends PureComponent {
       ...otherProps
     } = this.props;
     const { selectedIndex } = this.state;
-
     return (
-      <div>
-        <div className={headerContainer} style={customHeaderContainer}>
+      <div
+        className={`${orientation === 'vertical'
```

Comment:
I'm definitely a fan of doing the least amount of computation in the jsx as possible. What do you think about trying to assign variables above the `return` in the future?

Reply:
Yep, that makes sense!

Reply:
updated!